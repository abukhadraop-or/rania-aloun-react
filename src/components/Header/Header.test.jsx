// import React from 'react';
// import Enzyme, { shallow } from 'enzyme';
// import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
// import Header from '.';

// Enzyme.configure({ adapter: new Adapter() });

// describe('<Header />', () => {
//   const wrapper = shallow(<Header />);

//   it('matches snapshot', () => {
//     expect(wrapper).toMatchSnapshot();
//   });
// });
