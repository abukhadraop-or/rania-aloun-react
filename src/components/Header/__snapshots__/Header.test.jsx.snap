// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<styled.header>
  <styled.p>
    <Styled(Link)
      to="/"
    >
      conduit
    </Styled(Link)>
  </styled.p>
  <NavBar />
</styled.header>
`;
